.github/workflows/sphinx.yml

```diff
@@ -35,11 +36,10 @@ jobs:
       - name: Build Documentation
         run: make docs
```

Comment:
The idea to keep `make docs` in the main Makefile is simple - it's an easy way to let users build the docs.
They don't need to type `sphinx-build` command or remember it.

At the same time the process of updating source strings hasn't changed for a looooong time, so let's keep the command here, instead of putting it in Makefile.